Sources/TuistKit/Services/AuthTokenRefreshService.swift

```diff
@@ -0,0 +1,97 @@
+import FileSystem
+import Foundation
+import Mockable
+import Path
+import ServiceContextModule
+import TuistCore
+import TuistLoader
+import TuistServer
+import TuistSupport
+
+@Mockable
+protocol AuthTokenRefreshServicing: AnyObject {
+    func run(
+        directory: String?
+    ) async throws
```

Comment:
The `run` convention is only for services that serve as executors of commands. Any other service should have more explicit method names.